src/views/Deposit/DepositDetail.tsx

```diff
@@ -210,12 +259,14 @@ const DepositDetail: React.FC<{}> = ({}) => {
         name: match.params.assetName,
       }
       setAsset(assetInfo);
+      */
     }
-  })(); }, [match]);
+  })(); }, [match, asset, balance]);
```

Comment:
Asset doesn’t need to be here- most of this doesn’t, but I’m fine with it since this code is getting replaced as soon as we can make a better view flow. The important parts are the react-query bit, which are Good Enough(tm) for now.